examples/simple-rest/README_es.adoc

```diff
@@ -0,0 +1,140 @@
+:index-group: REST
+:jbake-type: page
+:jbake-status: status=published
+
+= REST Simple
+
+Definir un servicio Rest es muy fácil, simplemente necesitamos agregar la anotación ``@Path`` en la clase y después definir los métodos HTTP que vamos a usar (``@GET``, ``@POST``, …).
```

Comment:
I did not know you can use indistinctly single quote `  or 2 single quotes ``, interesting